bin/rubygems_encrypt_for_travis.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+if [ -z ${1} ]; then
+    echo "Usage: $0 <rubygems api key>"
+    exit 1
+fi
+
+rubygems_api_key=$1
+travis_args="--skip-version-check --skip-completion-check --no-interactive"
+
+for module in `cat share/modules`
```

Comment:
could you migrate this to while read -r line....
and maybe also to $() instead ob backticks?